packages/core/src/event-store/migration.test.ts

```diff
@@ -0,0 +1,186 @@
+import "@/utils/globals";
+
+import { hexToNumber, RpcBlock, RpcTransaction } from "viem";
+import { beforeEach, expect, test } from "vitest";
+
+import { setupEventStore } from "@/_test/setup";
+
+beforeEach(
+  async (context) => await setupEventStore(context, { skipMigrateUp: true })
+);
+test("setup creates tables", async (context) => {
+  const { eventStore } = context;
+
+  const tables = await eventStore.db.introspection.getTables();
+  const tableNames = tables.map((t) => t.name);
+  expect(tableNames).toContain("blocks");
+  expect(tableNames).toContain("contractReadResults");
+  expect(tableNames).toContain("logFilterCachedRanges");
+  expect(tableNames).toContain("logs");
+  expect(tableNames).toContain("transactions");
+});
+
+const blockOne: RpcBlock = {
```

Comment:
Would be great to avoid copy/pasting these RPC objects in both test files. You could move them over to `_test/constants.ts` and then import them in both `store.test.ts` and this file.